app/model/application.js

```diff
@@ -79,12 +87,16 @@ ApplicationSchema.statics.fetchReleases = function(application) {
 }
 
 ApplicationSchema.statics.parseAppHubFileIfPossible = function(application) {
-  return Promise.try(() => {
-    // Parse the .desktop file
-    const appHubFileBuf = new Buffer(application.appHubFileResult.content, 'base64');
-    const appHubData = JSON.parse(appHubFileBuf.toString());
-    application.priceUSD = appHubData.priceUSD;
-    delete application.appHubFileResult;
+  const fullName = application.github.fullName;
+  return gh.request(`GET /repos/${fullName}/contents/.apphub`, {
```

Comment:
ah, was this necessary because of the refactor I did in `dashboard.js` — no more `appHubFileResult`?

You may still want [`Promise.try()`](http://bluebirdjs.com/docs/api/promise.try.html), which rejects the promise if a synchronous error throws (namely `JSON.parse()` throwing if `.apphub` file has malformed JSON). Not sure what should happen exactly, though — if you reject the promise with .try(), the whole request will return a 500 error. Probably not what we want, it would be better to show a nice popover warning message or something. Either way, there should probably be some explicit error handling for that case... not necessary in this PR, but maybe file a bug as something to be worked on later.